src/Equinor.ProCoSys.IPO.WebApi/Controllers/Persons/PersonsController.cs

```diff
@@ -1,46 +1,67 @@
-﻿using System.Collections.Generic;
```

Comment:
why this difference on first 46 lines?
Is it some differences in setup of linebreaks as Pål mentioned in teams?

Reply:
File in Master is LF, commited with CRLF